README.md

```diff
@@ -13,18 +13,26 @@ JWST Calibration Pipeline
 Installation
 ------------
 
-The `jwst` package can be installed into a virtualenv or conda environment via `pip`.  We recommend creating a fresh environment with only python installed.  If using conda, make sure you have a recent version of Anaconda or Miniconda installed.  Via conda:
+The `jwst` package can be installed into a virtualenv or conda environment via `pip`.  In general, we recommend creating a fresh environment with only python installed.  If using conda, make sure you have a recent version of Anaconda or Miniconda installed.  From a bash shell enter:
 
-    conda create -n jwst_env python
-    conda activate jwst_env
+    conda create -n <env_name> python
+    conda activate <env_name>
+If conda is not in your path, you will need to cd to the bin directory of your anaconda installation and enter:
+
+    ./conda create -n <env_name> python
+    ./conda activate <env_name>
+
+## Below are directions on how to install three different versions of the pipeline.
+Note that you need to always create and activate your new evironment
+before proceeding.
 
 ### Installing for end-users ###
 
-To install a released (tagged) version, you can install directly from Github.  To install release `jwst 0.15.0`:
+If you want to run the latest fully tested version of the pipeline you should install a released (tagged) version. you can install these versions directly from Github.  For example, to install release `jwst 0.15.0`:
```

Comment:
Minor nit, but the latest tagged version is 0.15.1 (corresponding to B7.4.2), so I'd change 0.15.0 to 0.15.1

Reply:
I changed it to 0.15.1